Mathlib/Analysis/Fourier/Inversion.lean

```diff
@@ -0,0 +1,159 @@
+point `v`. (Morally, it converges to the Dirac at `v`). Moreover, it has integral one.
+Therefore, multiplying by `f` and integrating, one gets a term converging to `f v` as `c → ∞`.
+Since it also converges to `𝓕⁻ (𝓕 f) v`, this proves the result.
+
+To check the concentration property of the middle factor and the fact that it has integral one, we
+rely on the explicit computation of the Fourier transform of Gaussians.
+-/
+
+open Filter MeasureTheory Complex FiniteDimensional Metric Real Bornology
+
+open scoped Topology FourierTransform RealInnerProductSpace
+
+variable {V E : Type*} [NormedAddCommGroup V] [InnerProductSpace ℝ V]
+  [MeasurableSpace V] [BorelSpace V] [FiniteDimensional ℝ V]
+  [NormedAddCommGroup E] [NormedSpace ℂ E] {f : V → E}
+
+namespace Real
+
+lemma tendsto_integral_cexp_sq_smul (hf : Integrable f) :
+    Tendsto (fun (c : ℝ) ↦ (∫ v : V, Complex.exp (- c⁻¹ * ‖v‖^2) • f v))
+      atTop (𝓝 (∫ v : V, f v)) := by
+  apply tendsto_integral_filter_of_dominated_convergence _ _ _ hf.norm
+  · apply eventually_of_forall (fun v ↦ ?_)
+    nth_rewrite 2 [show f v = Complex.exp (- (0 : ℝ) * ‖v‖^2) • f v by simp]
+    apply (Tendsto.cexp _).smul_const
+    exact tendsto_inv_atTop_zero.ofReal.neg.mul_const _
+  · apply eventually_of_forall (fun c ↦ ?_)
+    exact AEStronglyMeasurable.smul (Continuous.aestronglyMeasurable (by continuity)) hf.1
+  · filter_upwards [Ici_mem_atTop (0 : ℝ)] with c (hc : 0 ≤ c)
+    apply eventually_of_forall (fun v ↦ ?_)
+    simp only [ofReal_inv, neg_mul, norm_smul, Complex.norm_eq_abs]
+    norm_cast
+    conv_rhs => rw [← one_mul (‖f v‖)]
+    gcongr
+    simp only [abs_exp, exp_le_one_iff, Left.neg_nonpos_iff]
+    positivity
+
+variable [CompleteSpace E]
+
+lemma tendsto_integral_gaussian_smul (hf : Integrable f) (h'f : Integrable (𝓕 f)) (v : V) :
+    Tendsto (fun (c : ℝ) ↦
+      ∫ w : V, ((π * c) ^ (finrank ℝ V / 2 : ℂ) * cexp (-π ^ 2 * c * ‖v - w‖ ^ 2)) • f w)
+    atTop (𝓝 (𝓕⁻ (𝓕 f) v)) := by
+  have A : Tendsto (fun (c : ℝ) ↦ (∫ w : V, cexp (- c⁻¹ * ‖w‖^2 + 2 * π * Complex.I * ⟪v, w⟫)
+       • (𝓕 f) w)) atTop (𝓝 (𝓕⁻ (𝓕 f) v)) := by
+    have : Integrable (fun w ↦ 𝐞[⟪w, v⟫] • (𝓕 f) w) := by
+      have B : Continuous fun p : V × V => (- innerₗ V) p.1 p.2 := continuous_inner.neg
+      simpa using
+        (VectorFourier.fourier_integral_convergent_iff Real.continuous_fourierChar B v).1 h'f
+    convert tendsto_integral_cexp_sq_smul this using 4 with c w
+    · rw [Real.fourierChar_apply, smul_smul, ← Complex.exp_add, real_inner_comm]
+      congr 3
+      simp only [ofReal_mul, ofReal_ofNat]
+      ring
+    · simp [fourierIntegralInv_eq]
+  have B : Tendsto (fun (c : ℝ) ↦ (∫ w : V,
+        𝓕 (fun w ↦ cexp (- c⁻¹ * ‖w‖^2 + 2 * π * Complex.I * ⟪v, w⟫)) w • f w)) atTop
+      (𝓝 (𝓕⁻ (𝓕 f) v)) := by
+    apply A.congr'
+    filter_upwards [Ioi_mem_atTop 0] with c (hc : 0 < c)
+    have I : Integrable (fun w ↦ cexp (- c⁻¹ * ‖w‖^2 + 2 * π * Complex.I * ⟪v, w⟫)) :=
```

Comment:
If `I` will just refer to the complex number, then `have I` should be updated a bit.